src/deepsparse/image_classification/schemas.py

```diff
@@ -44,15 +45,16 @@ class Config:
     @classmethod
     def from_files(cls, files: List[str], **kwargs) -> "ImageClassificationInput":
```

Comment:
Should the type of files be changed to `Iterable[TextIO]`?

Reply:
Good catch! thanks